docs/index.md

```diff
@@ -4,30 +4,49 @@ layout: home
+packages, tools, or services that are useful when building a Python package.
 Each entry has
 
-- a link to the package or service
-- a short summary of what it does
-- traffic lights
-- a link to a package developed by someone in ARC that uses it
+- a link to the package or service,
+- a short summary of what it does,
+- and traffic light!
 
 ## Choosing tools
 
-If you are working within a larger community, always start with the same tools they recommend.
-For example, the [`napari`](https://napari.org/) community have [`cookiecutter-napari-plugin`](https://github.com/napari/cookiecutter-napari-plugin) and the [SciKit-Surgery](https://scikit-surgery.github.io/scikit-surgery/) community have [PythonTemplate](https://github.com/SciKit-Surgery/PythonTemplate), both used to create new Python-based libraries.
-This makes it easier for others in the community to contribute to your package, and once you get used to the structure makes it easier for you to contribute back to other packages.
-
-Otherwise, each page on this site highlights recommended packages or services for each area.
-These should not be taken as set in stone for every project, but are a good place to start.
+If you are working within a larger community, always start with the same tools
+they recommend. For example, the [napari](https://napari.org/) community have a
+[`cookiecutter-napari-plugin`](https://github.com/napari/cookiecutter-napari-plugin)
+template and the
+[SciKit-Surgery](https://scikit-surgery.github.io/scikit-surgery/) community
+have [PythonTemplate](https://github.com/SciKit-Surgery/PythonTemplate), both
+used to create new Python-based libraries. Using common tooling and structure
+makes it easier for others in the community to contribute to your package. Once
+_you_ get used to the structure makes it easier for you to contribute back to
+other packages.
+
+🍪 If you just want to get started with our recommendations, we have our own
+[template](https://github.com/UCL-ARC/python-tooling#using-this-template) that
+lives in the same repository as these pages.
+
+Otherwise, each page on this site highlights recommended packages or services
+for each area. These should not be taken as set in stone for every project, but
+are a good place to start.
 
 ### Traffic lights
 
 Each item has an (opinionated) traffic light. The meaning of these is:
 
-- 🟢 At least one person in ARC uses this. We actively recommend using it above other tools. It is the single recommended package for a given purpose.
-- 🟠 We don't discourage using this, but it may duplicate functionality of a green tool.
-- 🔴 We actively discourage using this. This could be because it's no longer maintained, not open source, or difficult to use. Consider moving to alternatives if you're currently using something that's red. A reason for not using this is given.
+- 🟢 At least one person in ARC uses this. We actively recommend using it above
+  other tools. It is the single recommended package for a given purpose.
```

Comment:
```suggestion
  other tools. It is the single recommended tool for a given purpose.
```
Bit tangential as your changes are just in the formatting here I think, but I think _tool_ here is better than _package_ as it is more consistent with the previous usage in the bullet point and I think fits better with the wider range of non-package things we are giving traffic light recommendations on.